packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/doenet/text_input/state_vars/immediate_value.rs

```diff
@@ -4,93 +4,90 @@ use super::TextInputState;
 
 /// The dependencies of the immediate_value state variable of the text input component
```

Comment:
Docstrings need to be updated to the new vocabulary globally.

Reply:
Done